juwai/docker-images
# Create image for Amazon Linux

## Summary of changes

- Create image for Amazon Linux working with Vagrant
- Add PR template

## How to Test

1. Make sure you have latest Docker installed
1. Run `$ docker build -t juwaicom/amazonlinux-vagrant:1.0 amazonlinux-vagrant/`
